### PR TITLE
Hide inputs and related on cloud

### PR DESCRIPTION
**Pull request by Gaya**
Inputs on cloud will be managed through forwarders instead of `system/inputs`

Hide / disable features that allow access to these parts on cloud.

## Motivation and Context

See https://github.com/Graylog2/graylog-plugin-cloud/issues/99

## How Has This Been Tested?
Tested in the cloud environment

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring (non-breaking change)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.

## To do:
- [x] Disable inputs page and navigation
- [x] Solve notifications
- [x] Node Inputs
- [x] Input extractors

External:
- [x] Solve cloudwatch (https://github.com/Graylog2/graylog-plugin-integrations/pull/657)
- [x] Solve 0365 (https://github.com/Graylog2/graylog-plugin-enterprise-integrations/pull/425)

### CODE REVIEW

**Comment by Gaya** [comment]
@edmundoa this one is ready without the o365 and cloudwatch disabling. I think it's best to wait until cloud 4.0 lands until we handle those two case. I have two pending PRs open for it... but they need to be updated accordingly.

**Review by edmundoa — CHANGES_REQUESTED** [review]
This is looking quite good already. Besides the changed files, I think we are only missing one place in the UI where you can go to Inputs/Extractors:

Screen Recording 2020-11-20 at 13 33 02

File: graylog2-web-interface/src/logic/notifications/NotificationsFactory.jsx [hunk 1]
```diff
@@ -123,8 +124,10 @@ class NotificationsFactory {
             <span>
               Input {notification.details.input_id} has failed to start on node {notification.node_id} for this reason:
               »{notification.details.reason}«. This means that you are unable to receive any messages from this input.
-              This is mostly an indication for a misconfiguration or an error. You can click {' '}
-              <Link to={Routes.SYSTEM.INPUTS}>here</Link> to solve this.
+              This is mostly an indication for a misconfiguration or an error.
+              <HideOnCloud>
+                You can click <Link to={Routes.SYSTEM.INPUTS}>here</Link> to solve this.
+              </HideOnCloud>
```
[the remark this answers is not in the document]
Comment (Gaya) [reply graylog2-web-interface/src/logic/notifications/NotificationsFactory.jsx:130]: Good catch!

**Comment by Gaya** [comment]
> This is looking quite good already. Besides the changed files, I think we are only missing one place in the UI where you can go to Inputs/Extractors:

@edmundoa thanks! Didn't know about this one. Made another thing to filter it like we do the menu items.

**Review by edmundoa — APPROVED** [review]
Looks good to me :shipit:

**Pull request merged**